partials/specials/banners/podcast-committed.php

```diff
@@ -0,0 +1,109 @@
+<?php
+  $local_term = get_term_by( 'slug', 'committed', 'category' );
+  $base_image_path = get_stylesheet_directory_uri() . '/dist/img/specials/committed/';
+  $url = $local_term ? get_term_link( $local_term ) : false;
+if ( $local_term && $url ) {
+  ?>
+<div class="committed-banner__backgrounded" style="background-color:rgb(254, 105, 25); overflow: hidden; position: relative">
+  <style type="text/css">
+    .fallback .committed-banner__backgrounded {
+      background-image: url(<?php echo $base_image_path . 'committed-background.jpg'; ?>);
+    }
+    .webp .committed-banner__backgrounded {
+      background-image: url(<?php echo $base_image_path . 'committed-background.webp'; ?>);
+    }
+    .avif .committed-banner__backgrounded {
+      background-image: url(<?php echo $base_image_path . 'committed-background.avif'; ?>);
+    }
+    .committed-banner__backgrounded {
+      height: 267px;
+      background-size: cover;
+      background-position: 0% 60%;
+      background-repeat: no-repeat;
+    }
+    .committed-banner__grid-row{
+      justify-content: space-between;
+    }
+    .committed-banner__grid-item {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      padding: 0px;
```

Comment:
The padding probably shouldnt be removed from grid items as it will create vertical misalignments on the outside edges with other grid layouts on the page. It also means in the `m` scale of this banner the first and second grid elements directly touch on my screen which looks uncomfortable